webapp/src/components/CompetenciesPage.tsx

```diff
@@ -28,15 +28,15 @@ const CompetenciesPage = () => {
         <Grid item xs={12} md={8}>
           <Box>
             <CreateCompetencyForm
-              onCompetencyCreated={id =>
-                setNewlyCreatedCompetencyIds([...newlyCreatedCompetencyIds, id])
-              }
+              onCompetenciesChanged={id =>
```

Comment:
Interesting that `Competencies` here is plural, even though this form only changes one competency at a time.

Reply:
It should be singular.  I'll make that change.